hermes-client/src/main/java/pl/allegro/tech/hermes/client/restTemplate/RestTemplateHermesSender.java

```diff
@@ -66,7 +66,7 @@ private HermesResponse fromHttpStatusCodeException(HttpStatusCodeException excep
                 .build();
     }
 
-    private TreeMap<String, String> convertToCaseInsensitiveMap(Map<String, String> hashMap) {
+    public static TreeMap<String, String> convertToCaseInsensitiveMap(Map<String, String> hashMap) {
```

Comment:
you introduced dependency between WebClient and RestTemplate - why? do they really depend on each other? making this method public means it becomes part of a contract and can't be changed. as Go proverb says: "A little copying is better than a little dependency."

Reply:
You're right. I fixed it.